src/Raven.Client/Documents/BulkInsert/BulkInsertOperation.cs

```diff
@@ -281,8 +281,11 @@ private static bool CheckServerVersion(string serverVersion)
             if (serverVersion != null && Version.TryParse(serverVersion, out var ver))
             {
                 // Version 6 only from 6.0.2
+                if (ver.Major == 6 && ver.Minor > 0)
+                    return true;
                 if (ver.Major == 6 && ver.Build < 2)
```

Comment:
be more explicit here with the version, check also that Minor == 0

Reply:
Done